lib/github_check_run_service.rb

```diff
@@ -22,10 +22,14 @@ def run
     @conclusion = @report_adapter.conslusion(@report)
 
     result = {}
-    @annotations.each_slice(MAX_ANNOTATIONS_SIZE) do |annotation|
-      result.merge(client_patch_annotations(id, annotation))
-      # Don't need to merge twice
-      client_post_pull_requests(annotation[0])
+    if @annotations.empty?
```

Comment:
I know the test suite this action has inherited is a little clunky, but would you mind adding a test for this?

Reply:
It looks like you have added a test in f8ae251351b33d9e0a5a2ccad761566d3c335871. Thank you!